src/Autofac.Integration.Wcf/MatchingScopeLifetimeTags.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Autofac.Integration.Wcf
+{
+    /// <summary>
+    /// Well-known tags used in setting up matching lifetime scopes.
+    /// </summary>
+    public static class MatchingScopeLifetimeTags
+    {
+        /// <summary>
+        /// Tag used in setting up per-request lifetime scope registrations
+        /// (e.g., per-HTTP-request or per-API-request).
+        /// </summary>
+        public static readonly object WcfRequestLifetimeScopeTag = "WcfRequest";
```

Comment:
Let's make this `WcfInstanceContextLifetimeScopeTag` and `WcfInstanceContext`. Since the instance context is based on the life specified in the service, it's not always per-request. Update the comment accordingly, like:

```xml
<summary>
Tag used in setting up per-instance-context lifetime scope registrations. Note the
instance context is based on the service behavior so it may not be per-request.
</summary>
```

Reply:
Also note above comment about whether we should tag the scope at all. If scope tagging is removed, this also gets removed.

Reply:
I read the documentation again. I don't think I need the tag. I have an EntityClient that I want to be different per InstanceContext but used for every request down scope of an InstanceContext. If my implementation of IPerInstanceContextModuleAccessor registers EntityClient as SingleInstance(), if I understand correctly, it will only be a a singleton from the current scope and down. If that is correct, it should meet my need.

Reply:
You'll want to test that. I think that's right, but it'd be best to verify. That's a simple test.

Reply:
In ASP.NET Core they only provide instance-per-lifetime-scope, no "instance per request," and that seems to work fine. You may be able to sub in instance per lifetime scope and get the expected behavior.